### DISCUSSION TITLE
Support for Shelly 3EM

### DISCUSSION THREAD

**Discussion by higuelzim** [mongoose-os-apps/shelly-homekit#1496] [Ideas]
Hi please let me now if there is a flash firmware for shelly em to HomeKit. I didn't find it anywhere.

I appreciate 

Many thanks

**Comment by rojer** [comment]
not currently supported and unlikely to be soon, but fair enough.

**Comment by kloibs** [comment]
If this were exposed in the Eve app similar to a native Eve product showing a graph with historical consumption... 🤤
